packages/cli/src/parallel-tests/run-all-tests.ts

```diff
@@ -215,7 +215,12 @@ export const runAllTests = async ({
     client,
     testRunId: testRun.id,
     status: overallStatus,
-    resultData: { results },
+    resultData: {
+      results: results.map(
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        ({ screenshotDiffResults, ...coreResult }) => coreResult
+      ),
+    },
```

Comment:
Discussion for later: we may want to update which data we store where (`Replay, TestRun, ReplayDiff`).

Reply:
Yes -- or if the BE already has the full ReplayDiff etc., we could have the BE return this, and run-all-tests can pipe through in same format.